openverse_catalog/dags/providers/provider_api_scripts/museum_victoria.py

```diff
@@ -127,41 +127,44 @@ def _get_media_info(media_data):
         media_type = media.get("type")
         if media_type == "image":
             image_id = media.get("id")
-            image_url, height, width, filesize, filetype = _get_image_data(media)
             license_url = _get_license_url(media)
-            if image_url is None or image_id is None or license_url is None:
+            if image_id is None or license_url is None:
+                continue
+            image_url, height, width, filesize = _get_image_data(media)
+            if image_url is None:
                 continue
             creators = _get_creator(media)
+            license_info = get_license_info(license_url=license_url)
             image_data.append(
                 {
                     "image_id": image_id,
                     "image_url": image_url,
                     "height": height,
                     "width": width,
-                    "license_url": license_url,
+                    "filesize": filesize,
+                    "license_info": license_info,
                     "creators": creators,
                 }
             )
     return image_data
 
 
-def _get_image_data(media):
+def _get_image_data(
+    media: Dict,
+) -> Tuple[str | None, int | None, int | None, int | None]:
+    height, width, filesize = None, None, None
     media_data = {}
-    filetype = None
-    if "large" in media.keys():
-        media_data = media.get("large")
-    elif "medium" in media.keys():
-        media_data = media.get("medium")
-    elif "small" in media.keys():
-        media_data = media.get("small")
+    for size in ["large", "medium", "small"]:
+        if size in media:
+            media_data = media[size]
+            break
```

Comment:
We could use a [`for...else` block](https://book.pythontips.com/en/latest/for_-_else.html) here to return early!